src/components/FinalSection.js

```diff
@@ -0,0 +1,110 @@
+//   } else if (screenSize >= 667 && screenSize < 1024) {
+//     purpleBackgroundUrl = PurpleBackgroundTabletOne + ',' +
+//     PurpleBackgroundTabletTwo + ',' PurpleBackgroundTabletThree;
+//   } else {
+//     purpleBackgroundUrl = PurpleBackgroundDesktopOne
+//     PurpleBackgroundDesktopTwo, PurpleBackgroundDesktopThree, PurpleBackgroundDesktopFour;
+//   }
+//   console.log('background', screenSize)
+
+  return (
+    <div className="finalSection">
+      {/* <img className="purpleBackground" src={purpleBackgroundUrl}
+        alt="Purple background shape" /> */}
+      <div className="purpleBackgroundContainerMobile">
+        <img className="purpleBackground" src="../images/PurpleBackgroundMobile.svg" alt="Purple background shape" />
+      </div>
+      <div className="purpleBackgroundContainerTablet">
+        <img className="purpleBackgroundT1" src="../images/PurpleBackgroundTabletOne.svg" alt="Purple background shape" />
+        <img className="purpleBackgroundT2" src="../images/PurpleBackgroundTabletTwo.svg" alt="Purple background shape" />
+        <img className="purpleBackgroundT3" src="../images/PurpleBackgroundTabletThree.svg" alt="Purple background shape" />
+      </div>
+      <div className="purpleBackgroundContainerDesktop">
+        <img className="purpleBackgroundD1" src="../images/PurpleBackgroundDesktopOne.svg" alt="Purple background shape" />
+        <img className="purpleBackgroundD2" src="../images/PurpleBackgroundDesktopTwo.svg" alt="Purple background shape" />
+        <img className="purpleBackgroundD3" src="../images/PurpleBackgroundDesktopThree.svg" alt="Purple background shape" />
+        <img className="purpleBackgroundD4" src="../images/PurpleBackgroundDesktopFour.svg" alt="Purple background shape" />
+      </div>
+      <div className="testimonalsWrapper">
+        <h2 className="testimonialsTitle">Why Students Love Our Barre Classes</h2>
+        <p style={{ marginBottom: '16px' }}>&quot;The instructors are amazing, the atmosphere is supportive,
+            and the workouts are challenging yet fun!&quot;
+        </p>
+        <p style={{ marginBottom: '24px' }}>-Robyn Andersson, <span>Barre Basic Student</span></p>
+        <p style={{ marginBottom: '16px' }}>&quot;I can feel the muscles working and getting stronger.
+            I always feel energized at the end of the class&quot;
+        </p>
+        <p>-Paula Ray, <span>Barre Strong Student</span></p>
+      </div>
+      <div className="registerWrapper">
+        <div className="topRegister">
+          <h2 className="registerTitle">Register at Raise Studio</h2>
+          <h3 className="accountTitle">Create an Account to Book Barre Classes and Buy Credits.</h3>
+          <p className="accountText">Already have an account? <button type="button" className="logInButton">Login</button></p>
+        </div>
+        <div className="bottomFormRegister">
+          <form className="registrationForm">
```

Comment:
Really nice form! The responsiveness is on point! It would be extra nice if you got some sort of error-message if you typed wrong sort of input o none. There are lots of React libraries that can help with that.